src/Toolkit.Preview/Toolkit.Preview.WPF/UI/Controls/TableOfContents/TableOfContents.Theme.xaml

```diff
@@ -47,7 +47,7 @@
                 </Style>
               </CheckBox.Style>
             </CheckBox>
-            <TextBlock Text="{Binding Content.Name, Mode=OneWay}" Grid.Column="1" VerticalAlignment="Center" />
+            <TextBlock Text="{Binding Content.Name, Mode=OneWay}" Grid.Column="1" VerticalAlignment="Center" Margin="5,0" />
```

Comment:
Why the right margin? Should this just be `5,0,0,0` ?

Reply:
The right margin improves the appearance of the selection rectangle. You can see the difference in the selected item in the before and after screenshots.